docs/theme-specification.md

```diff
@@ -131,68 +160,55 @@ m: [ 0, 1, 2 ]
 }
 ```
 
-#### Media Queries
-
-For convenience and for use with other styling approaches, a `mediaQueries` scale derived from the `breakpoints` scale can be added to the theme object.
-
-```js
-breakpoints: [ '40em', '52em', '64em' ]
-
-mediaQueries: {
-  small: `@media screen and (min-width: ${breakpoints[0]})`,
-  medium: `@media screen and (min-width: ${breakpoints[1]})`,
-  large: `@media screen and (min-width: ${breakpoints[2]})`,
-}
-```
-
-### Key Reference
+### Key Reference / aliases
 
 The following is a list of theme object keys and their corresponding CSS properties.
 This list may be non-exhaustive.
 
 Theme Key         | CSS Properties
 ------------------|--------------
 `space`           | `margin`, `margin-top`, `margin-right`, `margin-bottom`, `margin-left`, `padding`, `padding-top`, `padding-right`, `padding-bottom`, `padding-left`, `grid-gap`, `grid-column-gap`, `grid-row-gap`
-`fontSizes`       | `font-size`
-`colors`          | `color`, `background-color`, `border-color`
-`fonts`           | `font-family`
-`fontWeights`     | `font-weight`
-`lineHeights`     | `line-height`
-`letterSpacings`  | `letter-spacing`
-`maxWidths`       | `max-width`
-`minWidths`       | `min-widths`
-`widths`          | `width`
-`maxHeights`      | `max-height`
-`minHeights`      | `min-height`
-`heights`         | `height`
-`borders`         | `border`, `border-top`, `border-right`, `border-bottom`, `border-left`
-`borderWidths`    | `border-width`
-`borderStyles`    | `border-style`
+`font.size`       | `font-size`
```

Comment:
I get the logic with nested objects, but also tend to favor flatter structures in cases like this to avoid what I call "premature organization" – that said this is directly from CSS itself, so it could make sense

Reply:
Something also to consider is that if `font` is omitted entirely from a theme, tooling now needs to perform existence checks.

Reply:
Additionally, "merging" themes in tooling becomes more complex.